docs/source/schema/directives.md

```diff
@@ -29,16 +29,37 @@ For example, here's the GraphQL spec's definition of the `@deprecated` directive
 ```graphql
 directive @deprecated(
   reason: String = "No longer supported"
-) on FIELD_DEFINITION | ENUM_VALUE
+) on FIELD_DEFINITION | ARGUMENT_DEFINITION | INPUT_FIELD_DEFINITION | ENUM_VALUE
```

Comment:
cc @glasser

I did make the recommended update and agree with it, but it's worth noting that this is _not_ in the spec yet (even in the [working draft](https://spec.graphql.org/draft/#sec--deprecated), surprisingly, though it's implemented in `graphql-js`)!

Reply:
Yep, this is the current PR: https://github.com/graphql/graphql-spec/pull/805
We can leave it at the old version? I dunno.